lib/puppet/provider/network_config/redhat.rb

```diff
@@ -228,6 +228,6 @@ def self.unmunge(props)
   end
 
   def self.post_flush_hook(filename)
-    File.chmod(0644, filename)
+    File.chmod(0o644, filename)
```

Comment:
Style/NumericLiteralPrefix recommends this for octal stuff. I think it is okay because tests are green?